Configure.pl

```diff
@@ -851,6 +834,35 @@ sub wsl_bash_on_win {
     return ((readline $fh) =~ /\A\d\.\d\.\d-\d+-Microsoft\s*\z/) ? 1 : 0;
 }
 
+# set up cincludes and ldusr flags for a native library
+sub setup_native_libraries {
+    my $library = shift;
+    my $result_cflags = `$config{pkgconfig} --cflags $library`;
+    if ( $? == 0 ) {
+        $result_cflags =~ s/\n//g;
+        if (index($config{cincludes}, $result_cflags) == -1) {
+            $config{cincludes} .= ' ' if $config{cincludes};
+            $config{cincludes} .= $result_cflags;
+            print("Adding extra include for $library: $result_cflags\n");
+        }
+    }
+    else {
+        print("Error occured when running $config{pkgconfig} --cflags $library.\n");
+    }
+    my $result_libs = `$config{pkgconfig} --libs-only-L $library`;
+    if ( $? == 0 ) {
+        $result_libs =~ s/\n//g;
+        if (index($config{lincludes}, $result_libs) == -1) {
+            $config{lincludes} .= ' ' if $config{lincludes};
```

Comment:
This is better written as `$config{lincludes} = join(' ', $config{lincludes}, $result_libs);`